CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex

```diff
@@ -480,13 +480,23 @@ \subsection{Initiatives for future training programmes\label{sec:initiatives}}
 relative simplicity of the wikimedia-based toolset: users contribute their
 training material using just a web-browser, and in order to do this efficiently,
 the necessary learning curve has been kept appropriately shallow.
-
 An interesting exercise in this context has recently been made by colleagues of
 the GridKa School of computing: the material from this year (2017) has been made
 publicly available on WTL~\cite{WikiToLearnGridka}.
-This constitutes an intersting example of what can be accomplished using this
+This constitutes an interesting example of what can be accomplished using this
 platform; it is just a first example of what's possible, but an inspiring one.
 
+Another example of web-based platform is a collection of online tutorials
+(mostly written in French) hosted on the Gitlab IN2P3
+server~\cite{Code-Swim-Coaches}. Taking advantage of
+the Gitlab ability to host Docker images, those tutorials aim to avoid the
+"damned installation step" that often absorbs half of a training session.
```

Comment:
Need to fix quotes here for LaTeX, ``damned installation step''.

Reply:
Done.